jumpToPDF.py

```diff
@@ -12,10 +12,96 @@
 
 import sublime_plugin, os.path, subprocess, time
 
+# attempts to locate the sublime executable to refocus on ST if keep_focus
+# is set.
+def get_sublime_executable():
+	s = sublime.load_settings('LaTeXTools.sublime-settings')
+	plat_settings = s.get(sublime.platform(), {})
+	sublime_executable = plat_settings.get('sublime_executable', None)
+
+	if sublime_executable:
+		return sublime_executable
+
+	# we cache the results of the other checks, if possible
+	if hasattr(get_sublime_executable, 'result'):
+		return get_sublime_executable.result
+
+	# are we on ST3
+	if hasattr(sublime, 'executable_path'):
+		get_sublime_executable.result = sublime.executable_path()
+		return get_sublime_executable.result
+
+	# guess-work for ST2
+	startupinfo = None
+	shell = False
+	if sublime.platform() == 'windows':
+		startupinfo = subprocess.STARTUPINFO()
+		startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+		shell = True

```

Comment:
I'm happy to change it, but it works on my ST2 on Windows 7 this way. My main reason for setting `shell = True` on Windows is that it runs the command through `cmd.exe` which tends to behave more like I expect. I didn't expect it would affect the outcome here either way.